pkg/api/unversioned/types.go

```diff
@@ -23,8 +23,6 @@ package unversioned
 // TypeMeta describes an individual object in an API response or request
 // with strings representing the type of the object and its API schema version.
 // Structures that are versioned or persisted should inline TypeMeta.
-//
-// +k8s:deepcopy-gen=false
```

Comment:
were these removals intentional?

Reply:
Yes, they cause spurious rebuilds but generate no output (so the dep is never satisfied).